.buildkite/common.py

```diff
@@ -86,6 +88,31 @@ def pipeline_to_json(pipeline):
     return json.dumps(pipeline, indent=4, sort_keys=True, ensure_ascii=False)
 
 
+def get_changed_files(branch):
+    """
+    Get all files changed since `branch`
+    """
+    stdout = subprocess.check_output(["git", "diff", "--name-only", branch])
+    return [Path(line) for line in stdout.decode().splitlines()]
+
+
+def run_all_tests(changed_files):
+    """
+    Check if we should run all tests, based on the files that have been changed
+    """
+
+    # TBD:
+    return False
```

Comment:
Don't we need to delete this?

Reply:
We do

Reply:
yes, sorry. It was meant just to test. I am deleting this.